feature_tests/kotlin/somelib/src/test/kotlin/dev/diplomattest/somelib/NativeTest.kt

```diff
@@ -1,8 +1,6 @@
 package dev.diplomattest.somelib
 
 import org.junit.jupiter.api.Test
-import java.lang.foreign.MemorySession
-import java.lang.foreign.SegmentAllocator
```

Comment:
@emarteca the offending lines, if you want to try and remove these, maybe it'll work.